devlab/tests/test_resource_migration.py

```diff
@@ -157,6 +199,24 @@ def test_migrate_vms_parameters(self):
 
         src_vms = [vm for vm in src_vms if vm.__dict__['status'] != 'ERROR']
 
+        self.validate_resource_parameter_not_in_dst(
+            src_vms, dst_vms, resource_name='VM', parameter='name')
+        self.validate_resource_parameter_not_in_dst(
+            src_vms, dst_vms, resource_name='VM', parameter='config_drive')
+        self.validate_resource_parameter_not_in_dst(
+            src_vms, dst_vms, resource_name='VM', parameter='key_name')
+        # TODO:
+        #  Using relative paths is a bad practice, unfortunately this is the
+        #  only way at this moment.
+        #  Should be fixed by implementing proper package module for Cloud Ferry.
+        main_folder = os.path.dirname(os.path.dirname(os.getcwd()))
+        file_path = '{}/configs/filter.yaml'.format(main_folder)
```

Comment:
You're repeating yourself 3 times with the relative paths logic. Consider moving to a separate function.